README.md

```diff
@@ -2,11 +2,11 @@
 
 ## Description
 
-This charm creates self-hosted GitHub runners. Each unit of this charm will start a configurable number of LXD based containers and virtual 
+This machine charm creates self-hosted GitHub runners. Each unit of this charm will start a configurable number of LXD based containers and virtual
```

Comment:
"to host GitHub runners" -> "to host them"?

Reply:
fixed